src/generators/css-filter.ts

```diff
@@ -63,7 +73,10 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        let [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        if (hasRootIssue()) {
+            [r, g, b] = applyColorMatrix([0, 0, 0], createFilterMatrix(config));
```

Comment:
As far as I understand, for Light mode it should be `[255. 255, 255]`.

Reply:
This way worked for me is it has the rootIssue and set it to 0,0,0

Reply:
I Would test it 1 more time